### PR TITLE
Implement Krause synchronization attention and regression as classification

### PR DESCRIPTION
**Pull request by kaitlynchen**
Diff check

### CODE REVIEW

File: mvts_transformer/src/running.py
Comment (joshuafan) [inline mvts_transformer/src/running.py:601]: Proposed refactor (will add this later, didn't double-check yet, low-priority):

```
if not require_padding:  # I think the only difference of this branch is adding padding_masks to model, but it could just be None? Need to check
    padding_mask = None
result = self.model(X.to(self.device), padding_masks, plot_dir=plot_dir, return_embeddings=should_track_batch)
if isinstance(result, tuple) and len(result) == 4:
    raw_predictions, attn_weights_layers, attn_weights_pool, embeddings_layers = result
elif isinstance(result, tuple) and len(result) == 3:
    predictions, attn_weights_layers, attn_weights_pool = result
else:
    predictions = result

if self.use_hl_gauss:
    scalar_predictions = self.loss_module.predict(predictions)
else:
    scalar_predictions = predictions

if config['normalize_label']:
   scalar_predictions = scalar_predictions * config["label_std"] + config["label_mean"]
all_predictions.append(scalar_predictions.detach().flatten())
```

File: mvts_transformer/src/main.py [hunk 1]
```diff
@@ -261,6 +262,18 @@ def main(config):
...
 
+    if config["task"] == "regression" and config.get("regression_loss", "mse") == "hl_gauss":    
+        train_labels = my_data.labels_df.loc[train_indices].values
+        
+        if config["hl_gauss_min"] is None or config["hl_gauss_max"] is None:
+            if config.get("normalize_label", False):
+                config["hl_gauss_min"] = -5.0
+                config["hl_gauss_max"] = 5.0
+            else:
+                hl_min, hl_max = compute_hl_gauss_range(torch.tensor(train_labels), margin_factor=1.2)
```
Comment (joshuafan) [inline mvts_transformer/src/main.py:273]: Minor - but do we really need different logic for normalize_label, or could we just compute_hl_gauss_range on `(train_labels - config['train_mean']) / config['label_std']`?

Although now that I think about it - I don't think normalize_label is needed with the histogram loss, since the magnitude of the label doesn't influence the model's predictions - it's still just predicting probabilities of ~50 histogram bins

**Review by joshuafan — APPROVED** [review]
I'll merge this in for now. These comments are minor simplifications we could try in the future (they shouldn't impact correctness)
